app/forms/idv/profile_form.rb

```diff
@@ -23,7 +23,9 @@ def self.model_name
                         allow_blank: true
 
     delegate :user_id, :first_name, :last_name, :phone, :email, :dob, :ssn, :address1,
```

Comment:
alpha these too? looks like they are grouped logically, but at this point there are so many that it might be easier to read in alpha order...